gdb.cc

```diff
@@ -844,7 +844,7 @@ read_route()
         bounds.max_alt = FREAD_DBL;
       }
       bounds.min_lat = FREAD_LATLON;
-      bounds.min_lat = FREAD_LATLON;
+      bounds.min_lon = FREAD_LATLON;
```

Comment:
Now THAT is a gem of a find. This class of mistake actually has a name: the Last Line Effect. T